tests/integration/test_debug.py

```diff
@@ -359,3 +359,15 @@ def test_error_output_ddtracerun():
     stderr = p.stderr.read()
     assert b"DATADOG TRACER CONFIGURATION" not in stderr
     assert b"DATADOG TRACER DIAGNOSTIC - Agent not reachable" not in stderr
+
+
+def test_debug_span_log():
```

Comment:
Apart from `env`, this is similar to `tests.utils.call_program`. Perhaps adapt and use that?

Reply:
hmm I just followed the convention in the file 🤷

Reply:
Ah I see! Sorry, forget it then!